src/dlgPackageExporter.cpp

```diff
@@ -215,7 +254,17 @@ void dlgPackageExporter::slot_export_package()
         }
     }
 
-    writer.exportPackage(filePath);
+    if (!writer.exportPackage(mXmlPathFileName)) {
+        displayResultMessage(tr("Failed to export - could not write Mudlet items to %1.").arg(mXmlPathFileName),
+                             false);
+        // Although we have failed we must not just abort here we need to reset
```

Comment:
```suggestion
        // Although we have failed, we must not just abort here. We need to reset
```

Reply:
Will review and throw in some more commas/full stops if I can...